src/components/questions/ImageUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  ToastAndroid,
+  Alert,
+  PermissionsAndroid,
+} from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+const ImageUpload = () => {
+  const [image, SetImage] = React.useState('');
+
+  // Show toast alert messages
+  const setToastMsg = (msg: any) => {
+    ToastAndroid.showWithGravity(msg, ToastAndroid.SHORT, ToastAndroid.CENTER);
+  };
+
+  const uploadImage = () => {
+    launchImageLibrary({ mediaType: 'photo', quality: 1, includeBase64: true }, (response) => {
+      if (response.didCancel) {
+        setToastMsg('geen foto gekozen');
+      } else if (response.errorCode == 'permission') {
+        setToastMsg('permissie afgewezen');
+      } else if (response.errorCode == 'others') {
+        setToastMsg(response.errorMessage);
+      } else if (response.assets[0].fileSize > 2097152) {
+        Alert.alert('Maximum grootte verstreken', 'Selecteer een foto onder 2M B', [
```

Comment:
Waarom wordt hier een alert gebruikt en bij de andere een toast?

Reply:
hier werd een alert gebruikt omdat het meer tekst bevat. of ik had er een lange toast van kunnen maken. alert bleef dan vast staan als notification pop up totdat je het wegsluit. Maar de alert is verwijderd bij volgende commit